superdesk.config.js

```diff
@@ -0,0 +1,61 @@
+/**
+ *
+ * This is the default configuration file for the Superdesk application.
```

Comment:
imo would be better to by default get that file from `cwd`, if it's not there get it from core. so we don't have to use that --config flag all the time when working on superdesk repo